api/management/commands/acquiredata.py

```diff
@@ -27,3 +27,13 @@ def handle(self, *args, **options):
         if not os.path.exists(sample_path):
             sample_url = 'https://raw.githubusercontent.com/cognoma/cancer-data/master/data/samples.tsv'
             urlretrieve(sample_url, sample_path)
+
+        gene_path = os.path.join(options['path'], 'genes.tsv')
+        if not os.path.exists(gene_path):
+            gene_url = 'http://www.stephenshank.com/genes.tsv'
+            urlretrieve(gene_url, gene_path)
+
+        mutation_path = os.path.join(options['path'], 'mutation-matrix.tsv.bz2')
+        if not os.path.exists(mutation_path):
+            mutation_url = 'https://ndownloader.figshare.com/files/5864862'
```

Comment:
I just assigned [cognoma/cognoml#15](https://github.com/cognoma/cognoml/issues/15) to myself to help move this forward. We can definitely use our data retrieval code in other places, @stephenshank check if the code [here](https://github.com/cognoma/cognoml/blob/33045e853e57e18c06f64f6ab499da3081080a5a/cognoml/data.py#L60-L75) works for what you need. I'm happy to add any additional helper code as well, just let me know what you're looking for.